README.md

```diff
@@ -72,59 +66,23 @@ version of Istio is 1.3 and you wish to introduce 1.4 which has been under devel
 
 #### When Istio source code is branched
 
-The documentation repo pulls content from the Istio source repos for inclusion in the published site.
-When the source repos are branched in preparation for a release, a few changes are needed in the
-documentation repo to track this:
-
-1. Switch to the **master** branch of the istio/istio.io repo and make sure everything is up to date.
-
-1. Edit the file `Makefile.core.mk` and change the `SOURCE_BRANCH_NAME` variable to the
-name of the newly created source branches (in this case `release-1.4`).
-
-1. Edit the file `data/args.yml` and set the `source_branch_name` field to the name of the newly created source
-branches (in this case `release-1.4`).
-
-1. Run `make update_all` in order to retrieve the latest material from the source repositories.
-
-1. Commit the previous edits to your local git repo and push your **master** branch to GitHub.
+Run `make prepare-1.4.0`, and that's it. This will grab the latest material from the new istio source branch.
```

Comment:
Your usage examples use 1.7.x. I think the example of going from 1.3 to 1.4 is old. I might renumber them so it's going from 1.6.x to1.7.x to be a little more current.

Reply:
Sounds good!